prowler/providers/kubernetes/services/rbac/rbac_minimize_pod_creation_access/rbac_minimize_pod_creation_access.py

```diff
@@ -0,0 +1,52 @@
+from prowler.lib.check.models import Check, Check_Report_Kubernetes
+from prowler.providers.kubernetes.services.rbac.rbac_client import rbac_client
+
+
+class rbac_minimize_pod_creation_access(Check):
+    def execute(self) -> Check_Report_Kubernetes:
+        findings = []
+        # Check ClusterRoleBindings for pod create access
+        for cr in rbac_client.cluster_roles:
+            report = Check_Report_Kubernetes(self.metadata())
+            report.namespace = "cluster-wide"
+            report.resource_name = cr.metadata.name
+            report.resource_id = cr.metadata.uid
+            report.status = "PASS"
+            report.status_extended = (
+                f"ClusterRole {cr.metadata.name} does not have pod create access."
+            )
+
+            for rule in cr.rules:
+                if (rule.resources and "pods" in rule.resources) and (
+                    rule.verbs and "create" in rule.verbs
+                ):
+                    report.status = "FAIL"
+                    report.status_extended = (
+                        f"ClusterRole {cr.metadata.name} has pod create access."
+                    )
+                    break
+            findings.append(report)
```

Comment:
Can you create a function here not to have this two times for each kind of role?